projects/remote_transceiver/inc/sailbot_db.h

```diff
@@ -2,29 +2,138 @@
 
 #include <google/protobuf/repeated_field.h>
 
-#include "mongocxx/client.hpp"
-#include "mongocxx/collection.hpp"
-#include "mongocxx/instance.hpp"
-#include "sensors.pb.h"
+#include <mongocxx/collection.hpp>
+#include <mongocxx/database.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/pool.hpp>
 
-constexpr auto COLLECTION_AIS_SHIPS    = "ais_ships";
-constexpr auto COLLECTION_BATTERIES    = "batteries";
-constexpr auto COLLECTION_DATA_SENSORS = "data_sensors";
-constexpr auto COLLECTION_GPS          = "gps";
-constexpr auto COLLECTION_WIND_SENSORS = "wind_sensors";
-constexpr auto COLLECTION_PATH         = "path";
+#include "sensors.pb.h"
+#include "waypoint.pb.h"
+
+// BSON document formats (from: https://ubcsailbot.atlassian.net/wiki/spaces/prjt22/pages/1907589126/Database+Schemas):
+
+// GPS
+// {
+//   latitude: decimal,
+//   longitude: decimal,
+//   speed: decimal,
+//   heading: decimal,
+//   timestamp: <year - 2000>-<month>-<day> <hour>:<minute>:<second>
+// }
+
+// Global Path
+// {
+//   waypoints: [
+//     {
+//       latitude: decimal,
+//       longitude: decimal
+//     }
+//   ],
+//   timestamp: <year - 2000>-<month>-<day> <hour>:<minute>:<second>
+// }
+
+// Local Path
+// {
+//   waypoints: [
+//     {
+//       latitude: decimal,
+//       longitude: decimal
+//     }
+//   ],
+//   timestamp: <year - 2000>-<month>-<day> <hour>:<minute>:<second>
+// }
+
+// AIS Ships
```

Comment:
Sorry, I was missing `rot` in the confluence document for AIS Ships. Do you mind adding it here?